components/ui/search/SearchCheckbox.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="check-filter" @click="toggle">
+    <button class="checkbox">
+      <CheckIcon v-if="activeFilters.includes(facetName)" />
+    </button>
+    <p>{{ displayName }}</p>
+  </div>
+</template>
+
+<script>
+import CheckIcon from '~/assets/images/utils/check.svg?inline'
+
+export default {
+  name: 'SearchCheckbox',
+  components: {
+    CheckIcon,
+  },
+  props: {
+    facetName: {
+      type: String,
+      default: '',
+    },
+    displayName: {
+      type: String,
+      default: '',
+    },
+    activeFilters: {
+      type: Array,
+      default() {
+        return []
+      },
+    },
+  },
+  methods: {
+    toggle() {
+      this.$emit('toggle', this.facetName)
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+.check-filter {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  margin: 3px 0.5rem 0 0.5rem;
+  font-size: 1rem;
+  letter-spacing: 0.02rem;
+}
+
+p {
+  padding: 0.2rem 0rem;
+  margin: 0;
+}
+
+.checkbox {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+
+  width: 1.5rem;
+  height: 1.5rem;
+
+  padding: 0;
+  margin: 0 0.5rem 0 0;
+
+  svg {
+    color: var(--color-brand-light);
+    stroke-width: 0.2rem;
+    height: 1.2rem;
+    width: 1.2rem;
+    flex-shrink: 0;
+  }
+}
+</style>
```

Comment:
Why is this a separate component (and an odd one too). If you want to make it separate, then make a checkbox component which can be used across the site. This is only used once

Reply:
You are right, it is pretty odd. It's a copy of `SearchFilter.vue` with some modifications. The `showVersions` array in the `mods.vue` file is already odd, it's an array that is treated like a bool. I could make the component work with `v-model` and move all the spacing over to the filters panel to make it more reusable.

Reply:
It's now reusable with 4c5f116